gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/OutputService.java

```diff
@@ -25,9 +32,37 @@ public OutputService(FileExporter fileExporter) {
         this.fileExporter = fileExporter;
     }
 
-    public String buildAndExportTtcDocument(SweData sweData, ExecutionResult result) {
+    public String buildAndExportTtcDocument(SweData sweData, ExecutionResult<SweDichotomyResult> result) {
         TtcDocument ttcDoc = new TtcDocument(result);
         InputStream inputStream = ttcDoc.buildTtcDocFile();
         return fileExporter.exportTtcDocument(sweData, inputStream);
     }
+
+    public String buildAndExportEsFrVoltageDoc(SweData sweData, ExecutionResult<SweDichotomyResult> result) {
+        return buildAndExportVoltageDoc(DichotomyDirection.ES_FR, sweData, result);
+    }
+
+    public String buildAndExportFrEsVoltageDoc(SweData sweData, ExecutionResult<SweDichotomyResult> result) {
```

Comment:
Could you set the direction as an argument so you dont have to repeat the method ?